managers/paranet-operations-manager.js

```diff
@@ -89,29 +85,37 @@ class ParanetOperationsManager {
             operatorRewardPercentage,
             incentivizationProposalVotersRewardPercentage,
         );
-        if(incentiveType === INCENTIVE_TYPE.NEUROWEB) {
-            const {contract, tokenId} = resolveUAL(paranetUAL);
+        if ([INCENTIVE_TYPE.NEUROWEB, INCENTIVE_TYPE.ERC20].includes(incentiveType)) {
+            const { contract, tokenId } = resolveUAL(paranetUAL);
+
+            const isNativeReward = incentiveType === INCENTIVE_TYPE.NEUROWEB;
+
+            const emissionMultiplier =
```

Comment:
Not that important, but we can move it to the inputService, we do it like that for percentages, so just to be consistent